CONTRIBUTING.md

```diff
@@ -22,19 +21,47 @@ See the [XAITK-Saliency README] file for additional information.
 
         $ git push origin HEAD -u
 
-5.  Visit the Kitware XAITK-Saliency Github, browse to the "Pull requests" tab
+5.  Visit the Kitware XAITK-Saliency GitHub, browse to the "Pull requests" tab
     and click on the "New pull request" button in the upper-right.
     Click on the "compare across forks" link, browse to your fork and browse to
     the topic branch to submit for the pull request.
     Finally, click the "Create pull request" button to create the request.
 
 
-XAITK-Saliency uses GitHub for code review and Github Actions for continuous
-testing as new pull requests are made.
-All checks/tests must pass before a PR can be merged.
+XAITK-Saliency uses GitHub for code review and GitHub Actions for continuous
+testing.
+New pull requests trigger CI when the merge target is the `master` branch.
+All checks/tests must pass before a PR can be merged by an individual with the
+appropriate permissions.
+
+We use Sphinx for manual and automatic API [documentation](docs).
+
 
-Sphinx is used for manual and automatic API [documentation].
+## Class Naming Philosophy
+A class that embodies performing a central action should be named following the
```

Comment:
Valid, I'll incorporate some of that less-verbosity. I didn't want to use "function" so as to avoid using overloaded terminology (e.g. with programmatic functions `def ...(): ...` ).